Python/Product/PythonTools/PythonToolsService.cs

```diff
@@ -100,7 +100,7 @@ public sealed class PythonToolsService : IDisposable {
             _idleManager = new IdleManager(container);
             _advancedOptions = new Lazy<AdvancedEditorOptions>(CreateAdvancedEditorOptions);
             _debuggerOptions = new Lazy<DebuggerOptions>(CreateDebuggerOptions);
-            _experimentalOptions = new Lazy<Options.ExperimentalOptions>(CreateExperimentalOptions);
```

Comment:
Is this a deliberate choice to no longer have experimental options?

Reply:
Deliberate. We'll put future experimental options where they belong based on feature area, and put experimental suffix in the checkbox label.

Reply:
Also to work better side-by-side, we have to start using the same VS registry that the other options already use. To read/write via command line, we can use vsregedit.exe